### DISCUSSION TITLE
sixpair not working on version 22.

### DISCUSSION THREAD

**Discussion by takeshi-pumas** [HumanSupportRobot/QandA#33] [Q&A]
I havent been able to wirelessly pair my ps3 controller since 22 update. Anyone else had a similar issue?

**Comment by hsr-admin** [comment]
I am able to connect to the 22.10 release in HSRB without any problems.
Would the following be of any help?
https://docs.hsr.io/hsr_faq_en/hsr_common_faq.html#dualshock3
